test/models/customer_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+
+describe Customer do
+  describe "validations" do
+    before do
+      @customer = customers(:Kenji)
```

Comment:
This isn't working for a couple reasons. I've added a code suggestion to demonstrate what it should look like to work properly.
```suggestion
      @customer = customer(:customer_2)
```

Reply:
However, after updating this, I still get the following error for the validation test:
`NoMethodError: undefined method 'checked_out_count' for #<Array:0x00007f93a52c8130>`

Reply:
I'm guessing y'all were aware these were failing and just didn't have time to fix them.
In case you were unaware they were failing, I want to remind you that it's important to run tests after writing them to make sure they work and it's important to run all tests before submitted a pull request to make sure all tests pass.